src/10_functions.py

```diff
@@ -2,11 +2,25 @@
 
 # YOUR CODE HERE
 
+
+def is_even(nbr):
```

Comment:
You are duplicating a lot of code here and on line 23-26 try splitting the functions apart so each one does only 1 thing. Maybe change is_even to 
```
def is_even(num):
    if num % 2 == 0:
        return True
```
and build the second function around that.